pages/api/users.ts

```diff
@@ -22,6 +22,10 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse<
 				users = users.populate('application');
 			}
 
+			if (userType == 'JUDGE' && req.query.isCheckedIn) {
+				users = users.where('isJudgeCheckedIn').equals(true);
```

Comment:
i didn't know there was a `.where()` function in mongoose. TIL.

Reply:
I didnt know either, but copilot did lmao